de4dot.code/deobfuscators/Dotfuscator/Deobfuscator.cs

```diff
@@ -50,6 +50,7 @@ class Deobfuscator : DeobfuscatorBase {
 		//Options options;
 		string obfuscatorName = "Dotfuscator";
 
+		CflowDecrypter cflowDescrypter;
```

Comment:
Is this field needed? It's only used in one method and could be a local.

Reply:
ok,I will modify the codes.
thank you.

Reply:
ok.:)